_rules/active-focus-shortcut-1e9941.md

```diff
@@ -6,7 +6,7 @@ description: |
   This rule checks that keyboard shortcuts for user interface components using only printable characters are only available when the component has focus.
 accessibility_requirements: # Remove whatever is not applicable
   wcag21:2.1.4: # Character Key Shortcuts (A)
-    forConformance: true
+    forConformance: false
     failed: further testing needed
     passed: satisfied
     inapplicable: further testing needed

```

Comment:
We don't list the SC is the rule does not map to them (as in "fail-on-fail").
https://www.w3.org/TR/act-rules-format/#accessibility-requirements-mapping
> the rule must list all accessibility requirements from those documents that are not satisfied when one or more of the outcomes of the rule is failed.